torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -3806,8 +3926,8 @@ struct to_ir {
     return emitBuiltinCall(loc, *graph, aten::slice, args, {step_nv});
   }
 
-  // Desugars slice indexing: tensor[begin:end] -> tensor.slice(dim, begin, end,
-  // 1)
+  // Desugars slice indexing: tensor[begin:end] -> tensor.slice(dim, begin,
```

Comment:
Changes that start from here to end of file seem to be lint only, could you move them to another PR?

Reply:
Thanks, I'll remove them